Jenkinsfile

```diff
@@ -186,6 +186,8 @@ pipeline {
         }
 
         cleanup {
+            archiveArtifacts artifacts: '/tmp/infinispan-operator/*', followSymlinks: false
```

Comment:
This needs to be a blob that will recursively match all `.yaml` files in all subdirectories, so `/tmp/infinispan-operator/**/*.yaml`

Reply:
The method also generate .log files with server logs that wouldn't match this predicate. Obviously that's solvable by adding another predicate ending with `.log`, aren't we going to lose the directory structure representing the test if do that though?

Reply:
You can update the GLOB to include `.log` as well, or you can provide multiple GLOBs separated via a `,`.

I don't we'll lose the FS structure, as the GLOB is just doing the matching.

Reply:
@ryanemerson Updated. Btw, one more thing I'm confused about is that I don't see the artifacts in the build, do you know where we can find them? I'd expect them to be available on the overview of the build run. If the build didn't find any artifacts it would fail but that didn't happen